GWAS/LMM.ipynb

```diff
@@ -392,26 +392,30 @@
     "On a minimal working example (MWE) dataset,\n",
     "\n",
     "```\n",
-    "sos run LMM.ipynb regenie \\\n",
+    "sos run ~/bioworkflows/GWAS/LMM.ipynb regenie \\\n",
```

Comment:
Please don't include path to the notebook because paths can be different for different people. You can always create a symbolic link to the file , as I mentioned before I think and again here:

```
cd /path/to/where/you/want/to/run/analysis
ln -s ~/bioworkflows/GWAS/LMM.ipynb # so you create a symoblic link called LMM.ipynb in the current folder to the source file
sos run LMM.ipynb ...
```